src/ftest.jl

```diff
@@ -51,15 +60,15 @@ function ftest(mod1::LinPredModel, mod2::LinPredModel)
     nparams2 = dof(mod2)
 
     df2 = nparams1-nparams2
-    df1 = dof_residual(mod1)
+    df1 = Int(dof_residual(mod1))
 
     MSR1 = (SSR2-SSR1)/df2
     MSR2 = SSR1/df1
 
     fstat = MSR1/MSR2
     pval = ccdf(FDist(df2, df1), fstat)
 
-    return FTestResult(SSR1, SSR2, df1, df2, MSR1, MSR2, fstat, pval)
+    return FTestTable(SSR1, SSR2, df1, df2, MSR1, MSR2, fstat, pval)
 end
 
 function ftest(mods::LinPredModel...)
```

Comment:
The methods should be defined the other way around: you don't even need to have a special method for the two-argument case, it's just a special case of this one.